tools/hardhat-viem-example/README.md

```diff
@@ -0,0 +1,103 @@
+# Hedera Hardhat Viem Example Project
+
+This Hedera Hardhat Viem Example Project offers boilerplate code for testing and deploying smart contracts via Hardhat with Viem. It includes configuration for both community-hosted and local ([Hedera Local Node](https://github.com/hashgraph/hedera-local-node)) instances of the [Hedera JSON RPC Relay](https://github.com/hashgraph/hedera-json-rpc-relay). 
+
+:fire: Check out the step-by-step tutorial [here](https://docs.hedera.com/hedera/tutorials/smart-contracts/deploy-a-smart-contract-using-hardhat-and-hedera-json-rpc-relays).
+
+# Viem vs Ethers.js Comparison
+
+This comparison aims to assist developers in selecting the most suitable library for their specific project requirements. For further information about Viem, please visit [https://viem.sh/docs/introduction](https://viem.sh/docs/introduction).
```

Comment:
I would add somewhere that `viem` has already out-of-the-box config support for Hedera networks, defined in https://github.com/wevm/viem/blob/cc105f801ec69640d3d806d86b35e36002d8c912/src/chains/index.ts#L113-L115.

Reply:
Is something like this going to be enough, @acuarica https://github.com/hashgraph/hedera-json-rpc-relay/pull/2556/files#diff-f1d801da845a0917514e10450a28b4ab00e52fc9dca7a3ed8b5a0120340ffef0R19 ?